daemon/src/connection.rs

```diff
@@ -42,7 +42,12 @@ use xtras::SendInterval;
 pub const MAX_RECONNECT_INTERVAL_SECONDS: u64 = 60;
 
 const TCP_TIMEOUT: std::time::Duration = Duration::from_secs(10);
-const HANDLE_MESSAGE_TIMEOUT: std::time::Duration = Duration::from_secs(5);
+
+const HANDLE_CONTRACT_SETUP_TIMEOUT: std::time::Duration = Duration::from_secs(10);
+const HANDLE_ROLLOVER_MESSAGE_TIMEOUT: std::time::Duration = Duration::from_secs(10);
+const HANDLE_CURRENT_ORDER_TIMEOUT: std::time::Duration = Duration::from_secs(10);
+const HANDLE_SETTLEMENT_MESSAGE_TIMEOUT: std::time::Duration = Duration::from_secs(120);
+const HANDLE_PROTOCOL_MESSAGE_TIMEOUT: std::time::Duration = Duration::from_secs(60);
```

Comment:
🙃 Just pointing it out since I've just noticed it.

I think processing protocol messages by the ephemeral actors will be very fast (much faster than 60 seconds). The handler only has to forward the message to a task:

https://github.com/itchysats/itchysats/blob/82a255e9ab4205a3a10853c1f06cd0bffb89b37c/daemon/src/setup_taker.rs#L137-L145

cc @da-kami